docs/new-guide-how-to-manage-api-keys.md

```diff
@@ -0,0 +1,45 @@
+---
+id: new-guide-how-to-manage-api-keys
+title: How to Manage API Keys
+---
+
+:::note What is this guide about?
+This guide explains how to handle API keys to provide workspaces scoped API access to meshStack. Managing these keys effectively is crucial to maintain security and control over API access. 
+:::
+
+## Challenge
+
+You want to provide application teams dedicated API keys scoped for their workspaces to access only certain objects in meshStack. This is useful for integrations or automations that require API access without exposing broader permissions across the whole organization.
+
+## Prerequisites
+
+- API keys are enabled in the meshStack instance. Check in the admin area under settings > compliance > API keys.
+- No limiting quota set on the workspace for API keys. If you want to limit the number of API keys per workspace, please refer to [How to Limit Workspace Resources](new-guide-how-to-limit-workspace-resources.md).
+- Workspace owner or manager permissions on the workspace where you want to manage API keys.
```

Comment:
Organization Admins can also create API Keys in any workspace. But not sure if this should be mentioned in this guide.